sim/lib/mobilitysim.py

```diff
@@ -148,7 +148,7 @@ def _simulate_individual_real_trace(indiv, max_time, site_type, mob_rate_per_typ
         if worker_type==k:
             usual_sites_k.append(worker_work_site)
             work_site = worker_work_site
-        elif k == 4:
+        elif k == 5:
```

Comment:
My suggestion would be to replace all of these places with `if k == len(mob_rate_per_type)-1`, that way it will always be the last column, even if we add more site types

Reply:
I agree. This is better than hard-coded 5.